src/utils/version.js

```diff
@@ -0,0 +1 @@
+export const VERSION = '0.1.1';
```

Comment:
It could be possible if `/_api/` endpoint provides such information but currently no 😕